katello/katello-remove

```diff
@@ -6,7 +6,7 @@ echo "Important packages such as the following will be removed:"
 echo ""
 echo "  * httpd (apache)"
 echo "  * mongodb"
-echo "  * tomcat6"
+echo "  * tomcat"
```

Comment:
technically it is still `tomcat6` on rhel 6, but I think this update is more correct :cat2: